libs/crypto/src/lib.rs

```diff
@@ -399,6 +400,30 @@ enum Kdf {
     },
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub enum PasswordError {
+    Base64Decoding,
+    InvalidFormat,
+    InvalidKeyLength,
+    InvalidLength,
+    InvalidSaltLength,
+    NoMatchingAlgorithm(String),
```

Comment:
I don't think we need the string here as the other parts that parse this (such as the pw import) already supports getting out the algorithm and displaying it.